plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/AbstractPresentation.java

```diff
@@ -80,6 +86,36 @@ public void applyChanges() {
     @Override
     public void createPresentation(@NotNull final IResultSetController controller, @NotNull Composite parent) {
         this.controller = controller;
+        trackStatistics();
+    }
+
+    private void trackStatistics() {
```

Comment:
No, don't do that in here. See where `IResultSetPresentation#createPresentation` is called, and track it there